### DISCUSSION TITLE
How do I save this as an .xpi file? Normal methods not working.

### DISCUSSION THREAD

**Discussion by whybinpm** [argenos/zotero-mdnotes#98] [Q&A] [answered]
Hello,

I thought I would post here rather than raise as an issue, as I feel this may be down to user error.

However, I seem to be unable to save mdnotes as an .xpi file. I've tried using 3 different browsers, yet nothing works.

To download the .xpi file, I have been righclicking different links for the addon and selecing "Save link as...". I then go to rename it to .xpi. It does save it as an .xpi file, however I keep getting the following error:

"The add-on "%S" could not be installed. It may be incompatible with this version of Zotero."

![image](https://user-images.githubusercontent.com/84910658/119819170-fdd13080-bee7-11eb-9d97-87321b7e96dd.png)
Picture of the file I'm trying.

Am I doing something wrong? I did the exact same thing with the zotfile addon and it worked. However, I did right click and save straight from the Zotero plugin page for that addon.

Please let me know if you need any more info.

Thanks.

**Comment by whybinpm** [comment, marked as answer]
Ignore, I managed to find the xpi file here: https://github.com/argenos/zotero-mdnotes/releases/tag/0.1.3

So in case anyone else has this issue, the above link works. Just scroll down a bit.

**Comment by argenos** [comment]
In general (for future versions too), the link for the releases is the following: https://github.com/argenos/zotero-mdnotes/releases. If you are on Firefox, make sure to right-click on the `xpi` and `Save as`.
